plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixDistributedQueries.java

```diff
@@ -126,6 +127,17 @@ public void testInsert()
         assertUpdate("DROP TABLE test_insert");
     }
 
+    @Override
+    public void testInsertArray()
+    {
+        assertThatThrownBy(super::testInsertArray)
```

Comment:
can we use `try/finally` in `AbstractTestDistributedQueries#testInsertArray` to force drop tables since this now exits midway?

Reply:
i don't think it matters in this case (no shared resources used in the test), and try/finally doesn't help read the test code, so i would rather avoid it